notifyre.sh

```diff
@@ -46,23 +40,21 @@ notif_timer() {
   if [ "$elapsed" -gt "$MIN_INTERVAL" ]; then
     if [ "$(uname -s)" = "Darwin" ]; then
       # Use terminal-notifier for Mac OS
-      terminal-notifier -title "NotiFyre $(echo $command_status)"\
-        -subtitle "Command : $commandx"\
-        -message "Completed in $elapsed seconds"\
-        -timeout 5\
+      terminal-notifier -title "NotiFyre $(echo $command_status)" \
+        -subtitle "Command : $commandx" \
+        -message "Completed in $elapsed seconds" \
+        -timeout 5 \
         -closeLabel "Gotcha!"
     else
       # Use notify-send for others
-      notify-send "NotiFyre $(echo $command_status)" "Terminal in ${p[-2]}/${p[-1]} and exit $? \$" \
```

Comment:
You forgot to add the first quote "Terminal in ${p[-2]}/${p[-1]} and exit $? \$"

Reply:
I mentioned in the commit message, actually it isn't preferred
We need to have a small and precise notification, and that's why I removed that statement and the code relating to it

Reply:
The script is left pretty much open for configuration.

Let the message be there, if it does not suit someone, they can modify their local file.

Reply:
Well, okay.
I'm bringing back the message

Reply:
@kaustubhhiware any emphasis on the '$' after `exit $?`
I couldn't analyze its presence

Reply:
You can remove the` $?` part, it was merely for testing purposes. @athityakumar improved upon that by denoting the status by simple ticks and crosses.

Reply:
Good. I'm fine with it :+1: